index.html

```diff
@@ -614,9 +614,39 @@ <h2>
         " >
         <figcaption>
 Overview of DID architecture and the relationship of the basic components.
+See also: <a class="longdesc-link"
+href="#brief-architecture-overview-longdesc">narrative description</a>.
         </figcaption>
       </figure>
 
+      <div class="longdesc" id="brief-architecture-overview-longdesc">
+        <p>
+Six internally-labeled shapes appear in the diagram, with labeled arrows
+between them, as follows. In the center of the diagram is a rectangle labeled
+DID URL, containing small typewritten text "did:example:123/path/to/rsrc". At
+the center top of the diagram is a rectangle labeled, "DID", containing small
+typewritten text "did:example:123". At the top left of the diagram is an oval,
+labeled "DID Subject". At the bottom center of the diagram is a rectangle
+labeled, "DID document". At the bottom left is an oval, labeled, "DID
+Controller". On the center right of the diagram is a two-dimensional rendering
+of a cylinder, labeled, "Verifiable Data Registry".
+        </p>
+        <p>
+From the top of the "DID URL" rectangle, an arrow, labeled "contains", extends
+upwards, pointing to the "DID" rectangle. From the bottom of the "DID URL"
+rectangle, an arrow, labeled "refers, and
+<strong><em>dereferences</em></strong>, to", extends downward, pointing to the
+"DID document" rectangle. An arrow from the "DID" rectangle, labeled
+"<strong><em>resolves</em></strong> to", points down to the "DID document"
+rectangle. An arrow from the "DID" rectangle, labeled "refers to", points left
+to the "DID subject" oval. An arrow from the "DID controller" oval, labeled
+"controls", points right to the "DID document" rectangle. An arrow from the
+"DID" rectangle, labeled "recorded on", points downards to the right, to the
+"Verifiable Data Registry" cylinder. An arrow from the "DID document" rectangle
+points upwards to the right to the "Verifiable Data Registry" cylinder.
```

Comment:
```suggestion
"Verifiable Data Registry" cylinder. An arrow from the "DID document" rectangle,
labeled "recorded on", points upwards to the right to the "Verifiable Data
Registry" cylinder.
```

Reply:
Thanks @brentzundel; applied in a81a674b01cfd42f20b03e00420cc2b12eef908e.